src/core/metadata/parameters/wallet.rs

```diff
@@ -205,6 +205,73 @@ impl From<AuthorizationEncryptionEncValuesSupported> for Json {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct AuthorizationSignedResponseAlg(pub ssi::crypto::Algorithm);
+
```

Comment:
These new values should be in `verifier.rs` not `wallet.rs`, right?

Reply:
Nice call, and I realize you already had some of these implemented.

I've removed the extraneous implementations in [204fc40](https://github.com/spruceid/openid4vp/pull/43/commits/204fc40606f0b2389fdf728d885d1649f406b58b)